config.js

```diff
@@ -28,7 +28,8 @@ config.server = {
             value: 1544
         }
     ],
-    writeFormat: 'application-vnd-oma-lwm2m/text'
+    writeFormat: 'application-vnd-oma-lwm2m/text',
+    defaultAcceptFormat: 'text/plain'
```

Comment:
Each time a new configuration item is added to config.js we need in addition:

* Describe it at https://github.com/telefonicaid/lwm2m-node-lib#-configuration
* Define a env var variable for it (LWM2M_XXXX) and add it to the table of in the above section
* Modify the code to process the env var as equivalent way of providing configuration

Reply:
Fixed in ec7d1c7c2268bf6e4b0556c74667b36276e1436c